### DISCUSSION TITLE
Add Content Type Filter To Created View

### DISCUSSION THREAD

**Discussion by nthoman** [pnp/syntex-samples#7] [Ideas]
When a new Model is applied to a document library a view is created with the new fields. Could you also apply the filter to the content type in on these views? I have a document library that will have a few hundred Models applied to it. It's nice that you create the view, but not really useful since it's not filtered to the content type. I have to go back and edit or just delete every view you create.

Side note, it would also be nice if the view creation was optional. In some cases you might not needed it.

**Comment by jameseccles** [comment]
Thanks for the feedback!
